controllers/common/common.go

```diff
@@ -298,7 +308,7 @@ func (h *ErrorHandler) HandleReconcilerError(request reconcile.Request, in error
 		result = RetryNever
 		err = nil
 
-		h.Error(in, "waiting for host monitor", "request", request)
+		h.Info("monitor is running", "request", request)
```

Comment:
This message looks weird to me, Can we change it to "waiting for host monitor to trigger another reconciliation"?